app/models/utiac_decision.rb

```diff
@@ -0,0 +1,11 @@
+require "document_metadata_decorator"
+
+class UtiacDecision < DocumentMetadataDecorator
+  set_extra_field_names [
+    :country,
+    :country_guidance,
+    :decision_reported,
+    :judges,
+    :promulgation_date,
```

Comment:
Since this is such a domain specific term, is there any chance you could stick a comment next to it explaining what `promulgation` means?